### PR TITLE
Rely on default memory traits

### PR DESCRIPTION
**Pull request by tpadioleau**

### CODE REVIEW

**Comment by yasahi-hpc** [comment]
Resolves #166

**Review by yasahi-hpc — APPROVED** [review]
Thanks!
Could you please add a `docstring` for this?

File: common/src/KokkosFFT_traits.hpp [hunk 1]
```diff
@@ -244,9 +244,7 @@ using base_container_value_type = typename base_container_value<T>::value_type;
 template <typename T>
```
[the remark this answers is not in the document]
Comment (tpadioleau) [reply common/src/KokkosFFT_traits.hpp:244]: Sure

[the remark this answers is not in the document]
Comment (tpadioleau) [reply common/src/KokkosFFT_traits.hpp:245]: Actually there was one docstring, which still seems to hold. What do you think ?

[the remark this answers is not in the document]
Comment (yasahi-hpc) [reply common/src/KokkosFFT_traits.hpp:244]: Sorry, it was not clear. I would like a docstring which reflects this change. 
Before, it defines a manageable view type (just removing unmanaged traits, while keeping other memory traits).
In the new version, it defines a managed view type from a managed/unmanaged view type (other memory traits are discarded).

[the remark this answers is not in the document]
Comment (tpadioleau) [reply common/src/KokkosFFT_traits.hpp:245]: Done in c0bacf636285e57056274b9d80385e7cfe7b2b4c

**Pull request merged**